tracer/src/Datadog.FleetInstaller/ReturnCode.cs

```diff
@@ -17,4 +17,5 @@ internal enum ReturnCode
     ErrorRemovingAppPoolVariables,
     ErrorRemovingNativeLoaderFiles,
     ErrorRemovingCrashTrackerKey,
+    ErrorReadingIisConfiguration,
```

Comment:
Can we set the value for `ErrorRemovingNativeLoaderFiles` explicitly so that it doesn't accidentally get changed?

Reply:
Yup, coming up in a separate PR 🙂